Fakebook.Profile/Fakebook.Profile.UnitTests/DomainTests/ProfileModelTests.cs

```diff
@@ -0,0 +1,305 @@
+﻿using System;
+
+using Fakebook.Profile.Domain;
+
+using Xunit;
+
+namespace Fakebook.Profile.UnitTests.DomainTests
+{
+    public class ProfileModelTests
+    {
+        #region EmailTests
+        /// <summary>
+        /// Check that emails can't be null.
+        /// </summary>
+        [Fact]
+        public void EmailCantBeNull()
+        {
+            //arrange
+            string email = null;
+            //act
+            //assert
+            Assert.Throws<ArgumentNullException>(() => new DomainProfile(email));
+        }
+
+        /// <summary>
+        /// Check that invalid emails throw an exception.
+        /// </summary>
+        /// <param name="email">The email to test. Should be invalid.</param>
+        [Theory]
+        [InlineData("test1@test")]
+        [InlineData("test2.test2")]
+        [InlineData("test3.test3@test3")]
+        [InlineData("test4")]
+        public void InvalidEmailThrowsException(string email)
+        {
+            //arrange
+            //act
+            //assert
+            Assert.Throws<ArgumentException>(() => new DomainProfile(email));
```

Comment:
we probably want `Assert.ThrowsAny` in almost all situations instead of `Throws`